JASP-Engine/JASP/R/binomialtest.R

```diff
@@ -187,13 +198,13 @@ BinomialTest <- function(jaspResults, dataset = NULL, options, ...) {
     
   # Add footnote: Alternative hypothesis
   if (options$hypothesis == "lessThanTestValue")
-    note <- gettext("For all tests, the alternative hypothesis specifies that the proportion is less than ")
+    note <- gettextf("For all tests, the alternative hypothesis specifies that the proportion is less than %s.", options$testValueUnparsed)
```

Comment:
Shouldnt the option be called `testFormula` or something?

Reply:
The analysis should not have to parse the formula at the beginning of the analysis: this should be done automatically in the next JASP version. The analysis should just use the name testValue and should not worry about the kind of field the interface uses.